client/src/components/reviews/breakdownComponents/RatingBreakdown.jsx

```diff
@@ -1,12 +1,31 @@
 import React from 'react';
+import {BarGraph} from './BarGraph.jsx';
+import {BarGraphContainer, RecommendationContainer} from '../sharedStyles/sharedStyledComponents';
+
+
+let RatingBreakdown = ({ starRatings, totalCount, recommended }) => {
+
+  const recommendation = 100*recommended/totalCount;
+
+  const findPercent = (star) => {
+    return 100*starRatings[star]/totalCount;
+  }
+
+  const percent = {
+    5: findPercent(5),
+    4: findPercent(4),
+    3: findPercent(3),
+    2: findPercent(2),
+    1: findPercent(1)
+  }
 
-let RatingBreakdown = () => {
   return (<div>
-    <p><a>5 Stars</a> Graph </p>
-    <p><a>4 Stars</a> Graph </p>
-    <p><a>3 Stars</a> Graph </p>
-    <p><a>2 Stars</a> Graph </p>
-    <p><a>1 Stars</a> Graph </p>
+    <RecommendationContainer><a>5 Stars</a> <BarGraphContainer><BarGraph percent={percent[5]}/></BarGraphContainer> <p>({starRatings[5]})</p></RecommendationContainer>
+    <RecommendationContainer><a>4 Stars</a> <BarGraphContainer><BarGraph percent={percent[4]}/></BarGraphContainer> <p>({starRatings[4]})</p></RecommendationContainer>
+    <RecommendationContainer><a>3 Stars</a> <BarGraphContainer><BarGraph percent={percent[3]}/></BarGraphContainer> <p>({starRatings[3]})</p></RecommendationContainer>
+    <RecommendationContainer><a>2 Stars</a> <BarGraphContainer><BarGraph percent={percent[2]}/></BarGraphContainer> <p>({starRatings[2]})</p></RecommendationContainer>
+    <RecommendationContainer><a>1 Stars</a> <BarGraphContainer><BarGraph percent={percent[1]}/></BarGraphContainer> <p>({starRatings[1]})</p></RecommendationContainer>
+    <p>{recommendation.toFixed(0)}% of reviewers recommend this product</p>
```

Comment:
TWould it be possible to map over this data? Definitely a backlog ticket, but worth putting on trello possibly.

Reply:
Yup, I put it on Trello. I just figured I had a working product and I was going to finish the basic requirements before going back to refactor